docroot/modules/custom/foia_webform/src/FoiaSubmissionServiceApi.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\file\Entity\File;
+use Drupal\node\NodeInterface;
+use Drupal\webform\WebformInterface;
+use Drupal\webform\WebformSubmissionInterface;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\RequestException;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class FoiaSubmissionServiceApi.
+ */
+class FoiaSubmissionServiceApi implements FoiaSubmissionServiceInterface {
+
+  /**
+   * The Agency Lookup service.
+   *
+   * @var \Drupal\foia_webform\AgencyLookupService
+   */
+  protected $agencyLookUpService;
```

Comment:
Should probably spell `LookUp` with a lower case `u` for consistency with the actual class: `AgencyLookupService`